packages/angular-cli/models/webpack-build-development.ts

```diff
@@ -1,14 +1,3 @@
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-
 export const getWebpackDevConfigPartial = function(projectRoot: string, appConfig: any) {
-  return {
-    output: {
-      filename: '[name].bundle.js',
-      sourceMapFilename: '[name].bundle.map',
-      chunkFilename: '[id].chunk.js'
-    },
-    plugins: [
-      new ExtractTextPlugin({filename: '[name].bundle.css'})
-    ]
-  };
+  return { };
```

Comment:
If there's a reason for the file to exist then it shouldn't be removed. I didn't know of `NamedModulesPlugin` but a quick google search made me think it's main use is to have good names in HMR, is that correct? If so, it makes sense to have it automatically with HMR, yes.